plugins/experimental/maxmind_acl/mmdb.cc

```diff
@@ -498,8 +507,11 @@ Acl::eval_country(MMDB_entry_data_s *entry_data, const char *path, int path_len)
   bool ret     = false;
   bool allow   = default_allow;
   char *output = nullptr;
-  output       = static_cast<char *>(malloc((sizeof(char) * entry_data->data_size)));
+
+  // We need to null terminate the iso_code ourselves, they are unterminated in the DBs
+  output = (char *)malloc((sizeof(char) * (entry_data->data_size + 1)));
```

Comment:
can this be changed back to the static_cast?

Reply:
yea, these are just things im porting over from our 8.1.x branch for this as we've fixed things, and I think my editor may have stripped that out, Ill add it back